docs/api/stickyHeader.md

```diff
@@ -0,0 +1,60 @@
+# Sticky Header API
+
+The StickyHeader component interface allows you to create sticky headers within the user interface.
+
+## Import
+
+```js
+import { StickyHeader } from 'forcepoint-shared-components';
+```
+
+export type StickyHeaderProps = {
+  headerTitle?: string;
+  headerCtas: HeaderCta[];
+  showHeader: boolean;
+  linkComponent?: ElementType;
+  headerLogoAlt: string;
+};
+
+export type HeaderCta = {
```

Comment:
This section doesn't have code formatter, and not sure if we need it at all

Reply:
Yes, I left this by mistake, I was using it to create the table. Sorry! And good catch.